lib/base/tlsutility.cpp

```diff
@@ -179,7 +179,7 @@ Shared<boost::asio::ssl::context>::Ptr MakeAsioSslContext(const String& pubkey,
 
 	InitializeOpenSSL();
 
-	auto context (Shared<ssl::context>::Make(ssl::context::tlsv12));
+	auto context (Shared<ssl::context>::Make(ssl::context::tls));
```

Comment:
What happens here...

Reply:
`tlsv12` is TLS 1.2 and nothing else, `tls` is any TLS version, which is later restricted to 1.2+ (or 1.3+ if you configure it that way)